resources/views/site/home.blade.php

```diff
@@ -4,6 +4,10 @@
 Home
 @endsection
 
+@push('custom_header')
```

Comment:
I've never seen `@push` what does it do?

Reply:
'Pushes' the following block into the specified block - instead of overwriting it, just appends. In this case, the stylesheet code following is added into the 'custom_header' section, but doesn't overwrite whatever else might be there